src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -3340,5 +3340,146 @@ static void B(in object obj, int v)
 4
 ");
         }
+
+        [Fact, WorkItem(36856, "https://github.com/dotnet/roslyn/issues/36856")]
+        public void Crash36856()
```

Comment:
>Crash36856 [](start = 20, length = 10)

Consider using a name that is actually descriptive, without having to visit the linked bug.